frontend/pages/investor/[id]/index.tsx

```diff
@@ -0,0 +1,192 @@
+// import { useRouter } from 'next/router';
+import { useIntl } from 'react-intl';
+import { FormattedMessage } from 'react-intl';
+
+import { loadI18nMessages } from 'helpers/i18n';
+
+import ProfileHeader from 'containers/profile-header';
+import SDGs, { SDGType } from 'containers/sdgs';
+import TagsGrid, { TagsGridRowType } from 'containers/tags-grid';
+import { SocialType } from 'containers/website-social-contact';
+
+import Head from 'components/head';
+import LayoutContainer from 'components/layout-container';
+import { StaticPageLayoutProps } from 'layouts/static-page';
+import { PageComponent } from 'types';
+
+export async function getStaticProps(ctx) {
+  return {
+    props: {
+      intlMessages: await loadI18nMessages(ctx),
+    },
+  };
+}
+
+export async function getStaticPaths() {
+  return {
+    paths: [],
+    fallback: true,
+  };
+}
```

Comment:
This page will be SSR-ed so I think you should use the server functions instead.